Formula/m/mariadb@10.10.rb

```diff
@@ -1,10 +1,9 @@
 class MariadbAT1010 < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-10.10.5/source/mariadb-10.10.5.tar.gz"
-  sha256 "91cd4d426d38af09852ebdaf00e74211775ec6cc907f74c1aee0d45cdf5d996b"
-  license "GPL-2.0-only"
-  revision 1
+  url "https://archive.mariadb.org/mariadb-10.10.6/source/mariadb-10.10.6.tar.gz"
+  sha256 "e1e53011979d29f0ec26e13ab8caf613481bf029520566fb96817546103938d5"
+  license "GPL-2.0-or-later"
```

Comment:
Corrected the license; see https://github.com/MariaDB/server/blob/mariadb-10.10.6/COPYING#L299